tools/docker/docker-compile.pl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env perl
+
+# copyright: Maciej Pasternacki
+# http://3ofcoins.net/2013/09/22/flat-docker-images/
+#
+# TODO: replace with something homemade
+
```

Comment:
U sure? We don't even have the copyright for this file, I pulled it in from that dude. He published it via gist without copyright note